### DISCUSSION TITLE
Allow anonymous viewing of CI logs

### DISCUSSION THREAD

**Discussion by jay** [community/community#67926] [Actions]
### Select Topic Area

Product Feedback

### Body

In the curl project we have a bunch of jobs that can't be viewed unless signed in to GitHub. Sometimes I am using public or shared computers where I will not sign in. All other CI services that we use allow viewing logs without signing in. My suggestion is allow anonymous viewing of GitHub Actions logs.

![signin](https://github.com/community/community/assets/965580/d2d5db05-4fde-4126-98e1-aa1778b33fb0)

/cc @bagder

**Comment by AdnaneKhan** [comment]
I think the main reason for this is to prevent bulk scraping of GHA run logs. At least with users, GitHub can monitor accounts that are scraping lots of run logs.

**Comment by Charlotte-br560** [comment]
Absolutely! Enabling anonymous viewing for GitHub Actions logs is a great suggestion, especially for user convenience on public or shared computers. This can improve accessibility and align with practices from other CI services. Your input is valuable, and I recommend exploring options similar to those in projects like Crawlbase to see how they manage CI logs accessibility. Appreciate your feedback!

**Reply by rafasmour** [reply]
ai?

**Comment by anarazel** [comment]
At the very least job summaries shouldn't be hidden from the summary view when not logged in.  Given that step names etc are all visible, I don't see a justification for hiding the summaries?
